crates/chain/src/tx_graph.rs

```diff
@@ -528,36 +491,36 @@ impl<A: Clone + Ord> TxGraph<A> {
             }
         }
 
-        for (anchor, txid) in additions.anchors {
+        for (anchor, txid) in changeset.anchors {
             if self.anchors.insert((anchor.clone(), txid)) {
                 let (_, anchors, _) = self.txs.entry(txid).or_insert_with(Default::default);
                 anchors.insert(anchor);
             }
         }
 
-        for (txid, new_last_seen) in additions.last_seen {
+        for (txid, new_last_seen) in changeset.last_seen {
             let (_, _, last_seen) = self.txs.entry(txid).or_insert_with(Default::default);
             if new_last_seen > *last_seen {
                 *last_seen = new_last_seen;
             }
         }
     }
 
-    /// Previews the resultant [`Additions`] when [`Self`] is updated against the `update` graph.
+    /// Previews the resultant [`ChangeSet`] when [`Self`] is updated against the `update` graph.
     ///
-    /// The [`Additions`] would be the set difference between `update` and `self` (transactions that
+    /// The [`ChangeSet`] would be the set difference between `update` and `self` (transactions that
     /// exist in `update` but not in `self`).
-    pub fn determine_additions(&self, update: &TxGraph<A>) -> Additions<A> {
-        let mut additions = Additions::default();
+    pub(crate) fn determine_changeset(&self, update: &TxGraph<A>) -> ChangeSet<A> {
```

Comment:
Nit: I would also make `determine_changeset` consume `update`.

```suggestion
    pub(crate) fn determine_changeset(&self, update: TxGraph<A>) -> ChangeSet<A> {
```